modules/virt-about-vms-and-boot-sources.adoc

```diff
@@ -2,19 +2,14 @@
 //
 // * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
 
-
 :_content-type: CONCEPT
 [id="virt-about-vms-and-boot-sources_{context}"]
+= About VMs and boot sources
 
-= About virtual machines and boot sources
-
-Virtual machines consist of a virtual machine definition and one or more disks that are backed by data volumes. Virtual machine templates enable you to create virtual machines using predefined virtual machine specifications.
+Virtual machines (VMs) consist of a VM definition and one or more disks that are backed by data volumes. VM templates enable you to create VMs using predefined specifications.
 
-Every virtual machine template requires a boot source, which is a fully configured virtual machine disk image including configured drivers. Each virtual machine template contains a virtual machine definition with a pointer to the boot source. Each boot source has a predefined name and namespace. For some operating systems, a boot source is automatically provided. If it is not provided, then an administrator must prepare a custom boot source.
+Every template requires a boot source, which is a fully configured disk image including configured drivers. Each template contains a VM definition with a pointer to the boot source. Each boot source has a predefined name and namespace. For some operating systems, a boot source is automatically provided. If it is not provided, then an administrator must prepare a custom boot source.
 
 Provided boot sources are updated automatically to the latest version of the operating system. For auto-updated boot sources, persistent volume claims (PVCs) are created with the cluster's default storage class. If you select a different default storage class after configuration, you must delete the existing data volumes in the cluster namespace that are configured with the previous default storage class.
 
-To use the boot sources feature, install the latest release of {VirtProductName}. The namespace `openshift-virtualization-os-images` enables the feature and is installed with the {VirtProductName} Operator.
-Once the boot source feature is installed, you can create boot sources, attach them to templates, and create virtual machines from the templates.
-
-Define a boot source by using a persistent volume claim (PVC) that is populated by uploading a local file, cloning an existing PVC, importing from a registry, or by URL. Attach a boot source to a virtual machine template by using the web console. After the boot source is attached to a virtual machine template, you  create any number of fully configured ready-to-use virtual machines from the template.
+To use the boot sources feature, install the latest release of {VirtProductName}. The namespace `openshift-virtualization-os-images` enables the feature and is installed with the {VirtProductName} Operator. Once the boot source feature is installed, you can create boot sources, attach them to templates, and create VMs from the templates.
```

Comment:
>  The namespace `openshift-virtualization-os-images` enables the feature and is installed with the {VirtProductName} Operator

I see this is existing text. But, it doesn't make sense to me. Do I install the boot sources feature or do I install the Operator and it installs the boot sources feature?

Reply:
The text is nonsense. I didn't write it. Will delete it.